pandas/io/parsers.py

```diff
@@ -509,7 +508,6 @@ def _read(filepath_or_buffer: FilePathOrBuffer, kwds):
     "skip_blank_lines": True,
 }
 
-
```

Comment:
add this back (this will make `flake8` fail, having two lines among top-level functions is required in PEP-8)